front-end/src/organisms/WorkspaceSettings.js

```diff
@@ -133,6 +133,7 @@ function WorkspaceSettings({ onChangeVisibility }) {
         </div>
 
         <div className={`${componentName}-header`}>{renderedName}</div>
+        <div className = {`${componentName}-spacer`}></div>
         <div className={`${componentName}-body`}>
```

Comment:
Accepted incoming changes from develop but produced so more failed frontend tests in Sidebar.test.js. Double check to make sure it's not just my side.